src/main/java/seedu/mark/model/util/SampleDataUtil.java

```diff
@@ -0,0 +1,59 @@
+package seedu.mark.model.util;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import seedu.mark.model.Mark;
+import seedu.mark.model.ReadOnlyMark;
+import seedu.mark.model.bookmark.Bookmark;
+import seedu.mark.model.bookmark.Name;
+import seedu.mark.model.bookmark.Remark;
+import seedu.mark.model.bookmark.Url;
+import seedu.mark.model.tag.Tag;
+
+/**
+ * Contains utility methods for populating {@code Mark} with sample data.
+ */
+public class SampleDataUtil {
+    public static Bookmark[] getSampleBookmarks() {
```

Comment:
In order to run the app, the URL has to be replaced with valid URL to pass the validation. After changing to valid URLs, I managed to run the app locally :)

I collected some random URLs you can just copy paste :P
```
    public static Bookmark[] getSampleBookmarks() {
        return new Bookmark[] {
            new Bookmark(new Name("Alex Yeoh"), new Url("https://www.google.com/"),
                new Remark("Blk 30 Geylang Street 29, #06-40"),
                getTagSet("friends")),
            new Bookmark(new Name("Bernice Yu"), new Url("https://www.youtube.com/"),
                new Remark("Blk 30 Lorong 3 Serangoon Gardens, #07-18"),
                getTagSet("colleagues", "friends")),
            new Bookmark(new Name("Charlotte Oliveiro"), new Url("http://www.nus.edu.sg/"),
                new Remark("Blk 11 Ang Mo Kio Street 74, #11-04"),
                getTagSet("neighbours")),
            new Bookmark(new Name("David Li"), new Url("https://en.wikipedia.org/"),
                new Remark("Blk 436 Serangoon Gardens Street 26, #16-43"),
                getTagSet("family")),
            new Bookmark(new Name("Irfan Ibrahim"), new Url("https://www.reddit.com/"),
                new Remark("Blk 47 Tampines Street 20, #17-35"),
                getTagSet("classmates")),
            new Bookmark(new Name("Roy Balakrishnan"), new Url("https://www.facebook.com/"),
                new Remark("Blk 45 Aljunied Street 85, #11-31"),
                getTagSet("colleagues"))
        };
    }
```